### ISSUE TITLE
GraphQL requests are cached and not invalidated when product is saved

### ISSUE THREAD

**Issue by dhorytskyi** [fastly/fastly-magento2#384]
Cached GraphQL requests are not invalidated after updating product.

**Steps**
1. Configure fastly as described in https://devdocs.magento.com/cloud/cdn/configure-fastly.html
2. Create a category
3. Create a product and assign it to the category from previous step
4. Perform following GraphQL query using a GET request
```
query{
  products(currentPage:1,pageSize:6,filter:{category_id:{eq:"3"}}){
    total_count,
    items{
      __typename
      id
      sku
      name
      price_range{
        minimum_price{
          final_price{
            value
          }
        }
      }
    }
  }
}
```
where "3" is id of the previously created category
5. Update product name or price from backend
6. Perform same GraphQL request

**Actual result**
Shows the old data from cache

**Expected result**
Should display updated contents

**Comment by vvuksan** [comment]
This would be an issue with Magento as the Fastly plugin only invalidates a list of X-Magento-Tags sent by Magento. If something is not being invalidated you should check the log of things that are being invalidated when there is a change. You can use webhooks to monitor what is being invalidated

https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/WEBHOOK-NOTIFICATIONS.md

You can find out what tags are being sent by Magento for a specific URL by supplying

-HFastly-Debug:1 

header to the curl invocation and comparing those two.

**Comment by dhorytskyi** [comment]
Hi @vvuksan , it's issue with Fastly module, I've created PR to fix it.

**Comment by dhorytskyi** [comment]
Was fixed in #386
